core/src/test/java/com/linecorp/armeria/internal/server/annotation/AnnotatedValueResolverTest.java

```diff
@@ -375,6 +414,8 @@ enum ValueEnum {
         VALUE1, VALUE2, VALUE3
     }
 
+    static class DummyService {}
```

Comment:
Maybe `AttrPrefix`?

Reply:
I see. i will change from `DummyService` to `AttributeTestDummyService`.